### PR TITLE
(re)added onChange and defaultValue to Textarea

### PR DESCRIPTION
**Pull request by tomatenschaf**
# Description

onChange and defaultValue props were missing in rendered <textarea /> since last refactoring of Textarea (about 22 day ago). 

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Added tests to avoid this bug in future.

# Checklist:

- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] I have not generated a new package version. (the maintainers will handle that)

### CODE REVIEW

**Comment by alextrastero** [comment]
Looking good! could you remove the changelog and lib folder if possible?

**Comment by alextrastero** [comment]
I will handle that when building a new version

**Comment by alextrastero** [comment]
ping

**Comment by alextrastero** [comment]
ping

**Comment by tomatenschaf** [comment]
I reverted my libfolder and changelog commits. Sorry, I didn't realized, an action was required.

File: src/Textarea.js [hunk 1]
```diff
@@ -55,7 +55,9 @@ const Textarea = ({
         placeholder={placeholder}
         id={id}
         value={value}
+        defaultValue={defaultValue}
         ref={textareaRef}
+        onChange={onChange}
```
Comment (alextrastero) [inline src/Textarea.js:60]: nice!, since we're using `...rest` how about delete L21 and L23 and that's it?

Comment (tomatenschaf) [reply src/Textarea.js:60]: I prefer the explicit list of parameters. `...rest` is for HTMLTextareaElement attributes, that have no special meaning in react or materialize.

Comment (alextrastero) [reply src/Textarea.js:60]: fair enough

**Comment by alextrastero** [comment]
Fixes #1115

**Pull request merged**